xarray/core/nputils.py

```diff
@@ -14,6 +13,17 @@
     _USE_BOTTLENECK = False
 
 
+def normalize_axis_index(data, axis):
```

Comment:
I added this because I noticed it when manually reverting the quantile change from https://github.com/pydata/xarray/pull/3713/files